library/Zend/Validator/Date.php

```diff
@@ -101,6 +107,24 @@ public function setFormat($format = self::FORMAT_DEFAULT)
     }
 
     /**
+     * @param boolean $allowNull
+     * @return $this
+     */
+    public function setAllowNull($allowNull)
+    {
+        $this->allowNull = $allowNull;
+        return $this;
+    }
+
+    /**
+     * @return boolean
+     */
+    public function allowNull()
+    {
+        return $this->allowNull;
+    }
+
```

Comment:
@coolms I personally disagree, as I think the proposed syntax reads better in conditionals:

```php
if ($date->allowNull()) {
    // ...
}
```

Since it's a method providing a boolean response, and will be used for conditionals, the focus should not be on whether it's _getting_ a value, but on what it returns.